src/aics-image-viewer/components/StyleProvider/index.tsx

```diff
@@ -286,6 +289,19 @@ const CssProvider = styled.div<{ $theme: AppTheme }>`
     }
   }
 
+  /* Let us use buttons with type="text" as purely semantic containers - don't bring any styling! */
+  .ant-btn-text {
+    width: unset;
+    height: unset;
+    padding: unset;
+
+    &:hover:not(:disabled),
+    &:focus-visible:not(:disabled) {
+      background-color: unset;
+      border-color: transparent;
+    }
+  }
```

Comment:
This `type` of button is currently only used for the "open channel settings" button. I think it's safe enough to assume that text buttons, if they're used again, will be used for the purpose described. Open to arguments as to why I should specialize this though.